sentinelone/plugin.spec.yaml

```diff
@@ -3,7 +3,7 @@ extension: plugin
 products: [insightconnect]
 name: sentinelone
 title: SentinelOne
-version: 3.1.0
+version: 3.2.0
```

Comment:
I think this needs to be a major version bump

Reply:
Bumped the version up to 4.0.0.